tidb-cloud/dedicated-external-storage.md

```diff
@@ -0,0 +1,255 @@
+
+If your source data is stored in Amazon S3 or Google Cloud Storage (GCS) buckets, before importing or migrating the data to TiDB Cloud, you need to configure cross-account access to the buckets. This document describes how to do this for TiDB Cloud Dedicated clusters.
+
+If you need to configure these external storages for TiDB Cloud Serverless clusters, see [Configure External Storage Access for TiDB Cloud Serverless](/tidb-cloud/serverless-external-storage.md).
+
+## Configure Amazon S3 access
+
+To allow TiDB Cloud to access the source data in your Amazon S3 bucket, you need to configure the bucket access for TiDB Cloud. You can use either of the following methods to configure the bucket access:
+
+- Use an AWS access key: use the access key of an IAM user to access your Amazon S3 bucket.
+- Use a Role ARN: use a Role ARN to access your Amazon S3 bucket.
+
+<SimpleTab>
+<div label="Role ARN">
+
+Configure the bucket access for TiDB Cloud and get the Role ARN as follows:
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), get the TiDB Cloud account ID and external ID of the target TiDB cluster.
+
+    1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
+
+        > **Tip:**
+        >
+        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.
+
+    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
+
+    3. On the **Import** page, select **Import data from S3**.
+
+        If this is your first time importing data into this cluster, select **Import From Amazon S3**.
+
+    4. On the **Import Data from Amazon S3** page, click the link under **Role ARN**. The **Add New Role ARN** dialog is displayed.
+
+    5. Expand **Create Role ARN manually** to get the TiDB Cloud Account ID and TiDB Cloud External ID. Take a note of these IDs for later use.
+
+2. In the AWS Management Console, create a managed policy for your Amazon S3 bucket.
+
+    1. Sign in to the AWS Management Console and open the Amazon S3 console at [https://console.aws.amazon.com/s3/](https://console.aws.amazon.com/s3/).
+    2. In the **Buckets** list, choose the name of your bucket with the source data, and then click **Copy ARN** to get your S3 bucket ARN (for example, `arn:aws:s3:::tidb-cloud-source-data`). Take a note of the bucket ARN for later use.
+
+        ![Copy bucket ARN](/media/tidb-cloud/copy-bucket-arn.png)
+
+    3. Open the IAM console at [https://console.aws.amazon.com/iam/](https://console.aws.amazon.com/iam/), click **Policies** in the navigation pane on the left, and then click **Create Policy**.
+
+        ![Create a policy](/media/tidb-cloud/aws-create-policy.png)
+
+    4. On the **Create policy** page, click the **JSON** tab.
+    5. Copy the following access policy template and paste it to the policy text field.
+
+        ```json
```

Comment:
Just temporarily remove "json" to fix the color issue of this page. I'll add "json" back before merging this PR.
![image](https://github.com/user-attachments/assets/d4e2ce36-5bb9-4b6c-98cb-a9ff67d85363)
````suggestion
        ```
````